src/components/Checkbox/BaseCheckbox.js

```diff
@@ -26,14 +30,15 @@ const propTypes = {
     /** Additional styles to add to checkbox button */
     style: stylePropTypes,
 
-    /** Callback that is called when mousedown is triggered. */
-    onMouseDown: PropTypes.func,
-
     /** A ref to forward to the Pressable */
     forwardedRef: PropTypes.oneOfType([
         PropTypes.func,
         PropTypes.shape({current: PropTypes.instanceOf(React.Component)}),
     ]),
+/** A function that is called when the box/label is pressed */
+    onPress: requiredPropsCheck,
+/** Callback that is called when mousedown is triggered. */
```

Comment:
Can we have a space between these please

Reply:
Done in the latest commit. Fixed the formatting 0 lint errors

Reply:
Done